### PR TITLE
Make the RMTK library installable

### PR DESCRIPTION
**Pull request by daniviga**
- Fixes a clash in the python package name since an `__init__.py` was existing in the repo top level folder (which is and should not be a python package)
- Makes the RMTK (just the python package _not_ the notebooks) installable using `pip install [-e]` or `python setup.py`, so the notebook can run without having to set manually the `PYTHONPATH` which was causing confusion to users (also due the issue above)
- Move the ipython notebook out from the python package. They are not python code/packages and cannot be installed. They are run (from RMTK.ipynb) as usual from the repo location

## TODO
- We still need to track the dependencies and add them to the `setup.py` (with a proper version range)
- Update README

How it works now
```bash
virtualenv foo
source ~/foo/bin/activate

git clone https://github.com/GEMScienceTools/rmtk.git
cd rmtk
# install the rmtk library
pip install -e rmtk
# install ipython and run the notebooks
pip install jupyter ipython notebook
jupyter notebook
```

### CODE REVIEW

**Comment by daniviga** [comment]
Anyone?

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+of the RMTK is to provide a suite of tools for the creation of seismic
+risk input models and for the post-processing and visualisation of 
+OpenQuake risk results.
+Copyright (C) 2015-2017 GEM Foundation
+"""
+
+setup(
+    name='rmtk',
+    version='1.0.0',
+    description="The main purpouse of the RMTK is to provide a suite of tools
```
Comment (raoanirudh) [inline setup.py]: *purpose

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                 for the creation of seismic risk input models and for the
+                 post-processing and visualisation of OpenQuake risk results."
```
Comment (raoanirudh) [inline setup.py]: comma missing after the description field

File: setup.py (hunk 1 above)
Comment (raoanirudh) [inline setup.py]: The multi-line description string requires either triple quotes or parantheses

**Review by raoanirudh — APPROVED** [review]
Apart from the three comments, lgtm

**Comment by daniviga** [comment]
Thanks @raoanirudh !

**Pull request merged**